### PR TITLE
Non-uk signposting - expanders and inset text updates

### PR DESCRIPTION
**Pull request by jenhadfield-dfe**
### Trello card
https://trello.com/c/kNwjmdy9/6238-implement-new-expanders-inset-component-to-signpost-non-uk-users

### Changes proposed in this pull request
Add signposting for non-uk users using our new expander component and update existing inset texts to the new purple inset

### CODE REVIEW

File: app/components/content/talk_to_us_component.html.erb
Comment (jenhadfield-dfe) [inline app/components/content/talk_to_us_component.html.erb:1]: @emmaachesongray I tried using an expander here below the box, but it was too separate and easy to miss. It also didn't work inside the box. So instead tried an inset box, but didn't want to repeat the content above, so just used the different info about phone number - see what you think

**Comment by mariosh-e** [comment]
Just 2 need to be changed: 
**How to choose course** = remove duplication content under Quals “If your application is successful, the training provider may be able to help you with applying for your visa. [Find out how to apply for your visa to train to teach in England(opens in new window)](https://www.find-postgraduate-teacher-training.service.gov.uk/).” as this is said below in the other content .

**Funding and support if you're a parent or carer** = title of expander is incorrect it currently  states if you’re disabled. Change to Funding and support if you're a parent or carer. Also remove “(unless you’ve applied to the Afghan or Ukraine resettlement schemes) for this one too”.

File: app/views/content/funding-and-support/if-youre-a-parent-or-carer.md [hunk 1]
```diff
@@ -23,10 +23,15 @@ keywords:
     - Grant
     - Parents
     - Carers
-inset_text:
+expander:
   international-content:
-    text: Most non-UK citizens without <a href="https://www.gov.uk/guidance/indefinite-leave-to-remain-in-the-uk">indefinite leave to remain in the UK</a> will not be eligible for this financial support. Find out about the <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees">financial support available for non-UK citizens</a>.
-    color: grey
+    title: funding and support if you're disabled
```
Comment (gemmadallmandfe) [inline app/views/content/funding-and-support/if-youre-a-parent-or-carer.md:28]:
```suggestion
    title: funding and support if you're a parent or carer
```

Comment (mariosh-e) [inline app/views/content/funding-and-support/if-youre-a-parent-or-carer.md:28]:
```suggestion
    title: funding and support if if you're a parent or carer
```

File: app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training-course.md [hunk 2]
```diff
@@ -21,7 +21,52 @@ keywords:
...
+    title: what you need to get onto the course
+    text: |-
+      <h3>Check your qualifications<h3>
+      <p>
+      If you have qualifications from outside the UK, you’ll need to show that they meet the standards set for teacher training in England.</p>
+      <p>
+      <a href="/non-uk-teachers/non-uk-qualifications">You can get help comparing English and international qualifications</a>.
+      </p>
+      <p>
+      If your application is successful, the training provider may be able to help you with applying for your visa. <a href="https://www.find-postgraduate-teacher-training.service.gov.uk/">Find out how to apply for your visa to train to teach in England</a>.
```
Comment (mariosh-e) [inline app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training-course.md:35]: This sentence to be removed as it is said below in the 'Right to work' section

File: app/components/content/talk_to_us_component.html.erb [hunk 3]
```diff
@@ -25,6 +25,13 @@
           <%= render ChatComponent.new %>
         </div>
       </div>
+        </div>
+      <div class="inset">
+          <%= render Content::InsetTextComponent.new(
+            header: "Non-UK citizens:",
+            text: "You can call us on <a href=\"tel:+448003892500\">+44 800 389 2500</a>. Calls will be charged at your country's standard rate.",
+            color: "purple-white"
```
Comment (emmaachesongray) [inline app/components/content/talk_to_us_component.html.erb:33]:
```suggestion
          <%= render Content::InsetTextComponent.new(
            header: "Non-UK citizens:",
            text: "You can call us on <a href=\"tel:+448003892500\">+44 800 389 2500</a>, or use the free live chat service. Calls will be charged at your country's standard rate.",
            color: "purple-white"
```

File: app/views/content/is-teaching-right-for-me/maths/_article.html.erb [hunk 4]
```diff
@@ -122,13 +124,15 @@
...
     </ul>
-    <p>
-      Non-UK citizens without indefinite leave to remain in the UK are unlikely to be eligible for funding.
-    </p>
     <p>
       Find out more about <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees">fees and financial support for non-UK trainee teachers</a>.
     </p>
-
+  <%= render Content::ExpanderComponent.new(
+    title: "bursaries and scholarships",
```
Comment (gemmadallmandfe) [inline app/views/content/is-teaching-right-for-me/maths/_article.html.erb:131]: @emmaachesongray I wonder if this title needs to be broader given that the information above is wider than just bursaries and scholarships - and the information in the expander/on the destination page is similarly broader?

(reword as needed!!)

```suggestion
    title: "funding your training",
```

File: app/views/content/is-teaching-right-for-me/get-school-experience.md [hunk 5]
```diff
@@ -27,6 +27,10 @@ quote:
     text: "I volunteered at a local primary school for 2 days a week and from the moment I walked into my first class, it felt very natural to be interacting with children."
     name: "Zainab, career changer"
     classes: quote--indent
+expander:
+  non-uk-get-school-experience:
+    title: get school experience
+    text: To get school experience in England, you'll need the right to work in the UK. This may be difficult if you have not started your training yet. However, you do not need classroom experience to train to teach in England.
```
Comment (gemmadallmandfe) [inline app/views/content/is-teaching-right-for-me/get-school-experience.md:33]:
```suggestion
    text: To request a placement in England through our get school experience service, you'll need the right to work in the UK. This may be difficult if you have not started your training yet. However, you do not need classroom experience to train to teach in England.  
```

does the right to work in England criteria just apply to our GSE service (I think probably)?

In which case, I would suggest rewording this (please reword as needed!)

File: app/views/content/steps-to-become-a-teacher/_step_3_consider_getting_experience.html.erb [hunk 6]
```diff
@@ -1,3 +1,9 @@
 <p>Experiencing life in a school can help you decide if teaching is right for you and who you want to teach. This is a good way to give you a taste of what the classroom is really like.</p>
 
+<%= render Content::ExpanderComponent.new(
+    title: "experience in a classroom in England",
+    text: "<p>To get experience in a school in England, you'll need the right to work in the UK. If you don't have experience in a classroom, you can still apply for teacher training.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/steps-to-become-a-teacher/_step_3_consider_getting_experience.html.erb:5]: As per the comments on the GSE page, I think this is a bit misleading - you need right to work if you are using our GSE service but is that true for all voluntary positions mentioned on the GSE page?

I wonder if we'd be better not having this expander on the steps page?

Also I think the title sets the expectation that they can get school experience?

File: app/views/content/funding-and-support/salaried-teacher-training.md [hunk 7]
```diff
@@ -22,6 +22,15 @@ keywords:
     - Salaried teacher training
     - Salaried teacher training course
     - Paid teacher training
+expander:
+  salaried-teacher-training:
+    title: salaried teacher training
+    text: |-
+      <p>
+      You can apply for a <a href="/funding-and-support/salaried-teacher-training">salaried teacher training course</a>. However, these courses are limited in number and very competitive. On this type of course, you will not pay a fee, and you can earn money while you train in a school.
```
Comment (gemmadallmandfe) [inline app/views/content/funding-and-support/salaried-teacher-training.md:30]:
```suggestion
      You can apply for a salaried teacher training course</a>. However, these courses are limited in number and very competitive. On this type of course, you will not pay a fee, and you can earn money while you train in a school. 
```

Link needs removing - we are linking to the page we are already on :-)

File: app/views/content/train-to-be-a-teacher/what-is-qts.md [hunk 8]
```diff
@@ -17,10 +17,25 @@ promo_content:
...
+      <p>
+      After 4 years, you'll need QTS to teach in many schools in England.</p>
+      <p>
+      <a href="/non-uk-teachers/teach-in-england-if-you-trained-overseas">Find out how to teach in England if you trained outside the UK</a>.
+      </p>    
+  get-qts-through-teacher-training:
+    title: get QTS through teacher training
+    text: |-
+      <p>
+      If live outside of the UK, you can either get <a href="/non-uk-teachers/teach-in-england-if-you-trained-overseas#english-qualified-teacher-status-qts">qualified teacher status (QTS)</a> or <a href="/non-uk-teachers/international-qualified-teacher-status">international qualified teacher status (iQTS)</a>.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/what-is-qts.md:35]:
```suggestion
      If you live outside the UK, you can either get <a href="/non-uk-teachers/teach-in-england-if-you-trained-overseas#english-qualified-teacher-status-qts">qualified teacher status (QTS)</a> or <a href="/non-uk-teachers/international-qualified-teacher-status">international qualified teacher status (iQTS)</a>.</p>
```

missing word

plus we usually say 'outside the UK' - but happy for you not to change this if you disagree!

File: app/views/content/is-teaching-right-for-me/who-do-you-want-to-teach.md [hunk 9]
```diff
@@ -62,6 +68,7 @@ Once you have QTS, you're qualified to teach in primary and secondary schools. H
 The best way to work out which age group you would like to teach is to get real life experience in a classroom. 
 
 $get-school-experience$
+$experience$
```
Comment (gemmadallmandfe) [inline app/views/content/is-teaching-right-for-me/who-do-you-want-to-teach.md:71]: I'm not sure this is quite right - it's after the CTA to the GSE service so might be missed

I also wonder whether we should be pointing people to the GSE page on GIT here rather than straight to the GSE service - let's discuss!

**Review by gemmadallmandfe — COMMENTED** [review]
@emmaachesongray This looks great! Have a few comments for you to consider

File: app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md [hunk 10]
```diff
@@ -20,6 +20,12 @@ fullwidth: true
 content:
   - content/train-to-be-a-teacher/assessment-only-route-to-qts/listing
   - content/train-to-be-a-teacher/assessment-only-route-to-qts/update-details
+expander:
```
[the remark this answers is not in the document]
Comment (jenhadfield-dfe) [reply app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md]: have changed to an inset component

File: app/views/content/is-teaching-right-for-me/computing/_article.html.erb [hunk 11]
```diff
@@ -87,8 +92,13 @@ image_2_alt: "",
 <li>a parent or carer</li>
 <li>disabled</li>
 </ul>
-<p>Non-UK citizens without indefinite leave to remain in the UK are unlikely to be eligible for funding.</p>
 <p>Find out more about <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees">fees and financial support for non-UK trainee teachers</a>.</p>
+<%= render Content::ExpanderComponent.new(
+  title: "bursaries and scholarships",
```
[the remark this answers is not in the document]
Comment (emmaachesongray) [reply app/views/content/is-teaching-right-for-me/computing/_article.html.erb]: agreed, made the change!

File: app/views/content/is-teaching-right-for-me/get-school-experience.md (hunk 5 above)
Comment (emmaachesongray) [reply app/views/content/is-teaching-right-for-me/get-school-experience.md:33]: right to work also applied to internships as well

File: app/views/content/train-to-be-a-teacher/what-is-qts.md (hunk 8 above)
Comment (emmaachesongray) [reply app/views/content/train-to-be-a-teacher/what-is-qts.md:35]: ah good spot. I've removed the QTS link as couldn't see any obvious non-uk content on QTS for trainees.

File: app/views/content/steps-to-become-a-teacher/_step_4_find_a_teacher_training_course.html.erb [hunk 12]
```diff
@@ -2,6 +2,12 @@
 
 <p>You need QTS to teach in most primary and secondary schools in England but you do not need a PGCE to teach.</p>
 
+<%= render Content::ExpanderComponent.new(
+    title: "find a teacher training course",
```
[the remark this answers is not in the document]
Comment (emmaachesongray) [reply app/views/content/steps-to-become-a-teacher/_step_4_find_a_teacher_training_course.html.erb:6]: agree
```suggestion
    title: "visa sponsorship",
```

File: app/views/content/steps-to-become-a-teacher/_step_7_apply_for_a_teaching_role.erb [hunk 13]
```diff
@@ -2,4 +2,10 @@
 
 <p>Schools start to advertise their vacancies from January.</p>
 
+<%= render Content::ExpanderComponent.new(
+    title: "apply for a teaching role",
```
[the remark this answers is not in the document]
Comment (emmaachesongray) [reply app/views/content/steps-to-become-a-teacher/_step_7_apply_for_a_teaching_role.erb:6]:
```suggestion
    title: "right to work in the UK",
```

File: app/views/content/is-teaching-right-for-me/physics/_article.html.erb [hunk 14]
```diff
@@ -82,8 +87,13 @@ You could inspire your pupils to consider rewarding professions, from tackling c
 a parent or carer</li>
 <li>disabled</li>
 </ul>
-<p>Non-UK citizens are eligible for bursaries and scholarships for physics teacher training courses.</p>
 <p>Find out more about <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees">fees and financial support for non-UK trainee teachers</a>.</p>
+<%= render Content::ExpanderComponent.new(
+  title: "bursaries and scholarships",
```
[the remark this answers is not in the document]
Comment (emmaachesongray) [reply app/views/content/is-teaching-right-for-me/physics/_article.html.erb:92]:
```suggestion
  title: "fund your training",
```

File: app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training-course.md [hunk 15]
```diff
@@ -21,7 +21,49 @@ keywords:
   - qts
   - Teach First
   - Postgraduate teaching apprenticeship
-
+expander:
+  what-you-need-to-get-onto-the-course:
+    title: what you need to get onto the course
+    text: |-
+      <h3>Check your qualifications<h3>
```
[the remark this answers is not in the document]
Comment (jenhadfield-dfe) [reply app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training-course.md]: have removed the headings

File: app/views/content/steps-to-become-a-teacher/_step_3_consider_getting_experience.html.erb (hunk 6 above)
Comment (emmaachesongray) [reply app/views/content/steps-to-become-a-teacher/_step_3_consider_getting_experience.html.erb:5]: remove

File: app/views/content/steps-to-become-a-teacher/_step_5_apply_for_teacher_training.html.erb [hunk 16]
```diff
@@ -4,4 +4,9 @@
 
 <p><a href="/how-to-apply-for-teacher-training">Get tips on making a great application</a> including finding the right references and writing a personal statement.</p>
 
+<%= render Content::ExpanderComponent.new(
+    title: "apply for teacher training",
+    text: "<p>Improve your application with <a href=\"/non-uk-teachers/train-to-teach-in-england-as-an-international-student\">our advice on applying from outside the UK.</a>.</p>",
```
[the remark this answers is not in the document]
Comment (emmaachesongray) [reply app/views/content/steps-to-become-a-teacher/_step_5_apply_for_teacher_training.html.erb:9]: change to apply to training first, then visa

File: app/views/content/is-teaching-right-for-me/computing/_article.html.erb [hunk 17]
```diff
@@ -87,8 +92,13 @@ image_2_alt: "",
 <li>a parent or carer</li>
 <li>disabled</li>
 </ul>
-<p>Non-UK citizens without indefinite leave to remain in the UK are unlikely to be eligible for funding.</p>
 <p>Find out more about <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees">fees and financial support for non-UK trainee teachers</a>.</p>
+<%= render Content::ExpanderComponent.new(
+  title: "funding your training",
+  text: "If you do not have indefinite leave to remain in the UK, you're unlikely to be eligible for funding.",
```
Comment (emmaachesongray) [inline app/views/content/is-teaching-right-for-me/computing/_article.html.erb:98]:
```suggestion
  text: "If you do not have <a href="https://www.gov.uk/guidance/indefinite-leave-to-remain-in-the-uk">indefinite leave to remain in the UK</a>, you're unlikely to be eligible for funding.",
```

File: app/views/content/is-teaching-right-for-me/maths/_article.html.erb [hunk 18]
```diff
@@ -122,13 +124,15 @@
...
-    <p>
-      Non-UK citizens without indefinite leave to remain in the UK are unlikely to be eligible for funding.
-    </p>
     <p>
       Find out more about <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees">fees and financial support for non-UK trainee teachers</a>.
     </p>
-
+  <%= render Content::ExpanderComponent.new(
+    title: "funding your training",
+    text: "If you do not have indefinite leave to remain in the UK, you're unlikely to be eligible for funding.",
```
Comment (emmaachesongray) [inline app/views/content/is-teaching-right-for-me/maths/_article.html.erb:132]:
```suggestion
    text: "If you do not have <a href="https://www.gov.uk/guidance/indefinite-leave-to-remain-in-the-uk">indefinite leave to remain in the UK</a>, you're unlikely to be eligible for funding.",
```

File: app/views/content/steps-to-become-a-teacher/_step_5_apply_for_teacher_training.html.erb [hunk 19]
```diff
@@ -4,4 +4,9 @@
 
 <p><a href="/how-to-apply-for-teacher-training">Get tips on making a great application</a> including finding the right references and writing a personal statement.</p>
 
+<%= render Content::ExpanderComponent.new(
+    title: "applying for training and visas",
+    text: "If you need a visa to train to teach in England, you must have a confirmed offer of a training place before you apply for your visa. You should only apply for courses that support student or skilled worker visa sponsorship.",
```
Comment (gemmadallmandfe) [inline app/views/content/steps-to-become-a-teacher/_step_5_apply_for_teacher_training.html.erb:9]: @emmaachesongray do we need to add signposting to the visas for trainees page here? probably a question for Amy

**Comment by gemmadallmandfe** [comment]
Policy have reviewed

File: app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md [hunk 20]
```diff
@@ -20,6 +20,13 @@ fullwidth: true
 content:
   - content/train-to-be-a-teacher/assessment-only-route-to-qts/listing
   - content/train-to-be-a-teacher/assessment-only-route-to-qts/update-details
+inset_text:
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md:23]: @emmaachesongray we should remove lines 23 to 29 for now given that you've removed the expander from appearing on the front end

File: app/views/content/train-to-be-a-teacher/teacher-degree-apprenticeships.md [hunk 21]
```diff
@@ -12,6 +12,10 @@ navigation: 20.40
 navigation_title: If you want to do a teaching apprenticeship
 navigation_description: A teacher degree apprenticeship (TDA) lets you work at a school and earn a salary while getting a degree and qualified teacher status (QTS).
 talk_to_us: false
+expander:
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/teacher-degree-apprenticeships.md:15]: @emmaachesongray We should remove this from the front matter given we are not using the expander on this page at the moment

File: app/views/content/how-to-apply-for-teacher-training/if-your-application-is-unsuccessful.md [hunk 22]
```diff
@@ -51,6 +51,8 @@ It’s worth looking at the feedback the course provider has given to understand
 
 There could be a number of reasons your application was unsuccessful:
 
+$visa-sponsorship$
```
Comment (gemmadallmandfe) [inline app/views/content/how-to-apply-for-teacher-training/if-your-application-is-unsuccessful.md:54]: @emmaachesongray I'm not sure this makes sense to have this as an expander - and particularly not this high up the page

I don't think this is better than what is already in production! Let's discuss

File: app/views/content/how-to-apply-for-teacher-training/if-your-application-is-unsuccessful.md [hunk 23]
```diff
@@ -35,6 +51,8 @@ It’s worth looking at the feedback the course provider has given to understand
...
```
Comment (gemmadallmandfe) [inline app/views/content/how-to-apply-for-teacher-training/if-your-application-is-unsuccessful.md:54]: @emmaachesongray I'm not sure this makes sense to have this as an expander - and particularly not this high up the page

I don't think this is better than what is already in production! Let's discuss

File: app/components/content/talk_to_us_component.html.erb [hunk 24]
```diff
@@ -25,6 +25,13 @@
           <%= render ChatComponent.new %>
         </div>
       </div>
+        </div>
+      <div class="inset">
+          <%= render Content::InsetTextComponent.new(
+            header: "Non-UK citizens:",
+            text: "You can call us on <a href=\"tel:+448003892500\">+44 800 389 2500</a>, or use the free live chat service. Calls will be charged at your country's standard rate.",
+            color: "purple-white"
```
Comment (gemmadallmandfe) [inline app/components/content/talk_to_us_component.html.erb:33]: @emmaachesongray The background of this inset on `/help-and-support` is white not purple? I'm happy for this to be fixed retrospectively if needs be

File: app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training-course.md [hunk 25]
```diff
@@ -21,7 +21,45 @@ keywords:
...
+  what-you-need-to-get-onto-the-course:
+    title: what you need to get onto the course
+    text: |-
+      <p>
+      If you have qualifications from outside the UK, you’ll need to show that they meet the standards set for teacher training in England. You can <a href="/non-uk-teachers/non-uk-qualifications">get help comparing English and international qualifications</a>.
+      </p>
+      <p>
+      You’ll need the right to work or study in the UK to do your teacher training in England.</p>
+      <p>
+      If you do not have the right to work in the UK, you should only apply to courses that have visa sponsorship available. You can filter by ‘visa sponsorship’ to <a href="/non-uk-teachers/visas-for-non-uk-trainees">find courses where visas can be sponsored.</a>
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training-course.md:34]: @emmaachesongray This is the wrong link - it's mixed up with line 37

File: app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training-course.md [hunk 26]
```diff
@@ -21,7 +21,45 @@ keywords:
...
+      </p>
+      <p>
+      If your application is successful, the training provider may be able to help you with applying for your visa. <a href="https://www.find-postgraduate-teacher-training.service.gov.uk/">Find out how to apply for your visa to train to teach in England</a>.
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training-course.md:37]: @emmaachesongray This is the wrong link - it's mixed up with line 34

File: app/views/content/funding-and-support/tuition-fee-and-maintenance-loans.md [hunk 27]
```diff
@@ -58,10 +69,17 @@ The maximum amount you may be eligible to borrow is:
...
-## If you come from outside England
+## Check your eligibility for a bursary or scholarship
+
+You may also be eligible for a tax-free bursary or scholarship, depending on the subject you're training to teach.
+
+You do not have to pay a bursary or scholarship back. You can get a bursary or scholarship as well as any tuition fee loan and maintenance loan.
+
+[Find out if you're eligible for a bursary or scholarship](/funding-and-support/scholarships-and-bursaries).
+
+## If you're from other UK countries or Crown Dependencies
```
Comment (gemmadallmandfe) [inline app/views/content/funding-and-support/tuition-fee-and-maintenance-loans.md:82]: @emmaachesongray I'm not sure `other UK countries` quite makes sense in this heading as we haven't mentioned England at all on this page - does this need changing?

```suggestion
## If you're from Wales, Scotland, Northern Ireland or the Crown Dependencies
```

File: app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training-course.md (hunk 26 above)
Comment (emmaachesongray) [reply app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training-course.md:37]: changed to the link to find as per live page.

File: app/components/content/talk_to_us_component.html.erb (hunk 24 above)
Comment (emmaachesongray) [reply app/components/content/talk_to_us_component.html.erb:33]: I think we should do this retrospectively and with Jen/Martyn. I'm not sure if Jen might've left it white for a reason like so the there's enough colour contrast?

Comment (gemmadallmandfe) [reply app/components/content/talk_to_us_component.html.erb:33]: Agree - I'll put a ticket in the backlog for next sprint to review when Jen is back

File: app/components/content/talk_to_us_component.html.erb [hunk 28]
```diff
@@ -25,6 +23,12 @@
           <%= render ChatComponent.new %>
         </div>
       </div>
+        </div>
+      <div class="inset">
+          <%= render Content::InsetTextComponent.new(
+            header: "Non-UK citizens:",
+            text: "You can call us on <a href=\"tel:+448003892500\">+44 800 389 2500</a>, or use the free live chat service. Calls will be charged at your country's standard rate.", color: "purple-white"
```
Comment (gemmadallmandfe) [inline app/components/content/talk_to_us_component.html.erb:30]: @emmaachesongray I'm not sure why `text` and `color` are on the same line now - they should really be on separate lines

However, I know you had linting issues around this so if you want to leave and get Jen to have a look when she considers the purple/white background issue when back from leave, that's fine with me as it looks OK on the front-end in the meantime

File: app/views/content/train-to-be-a-teacher/teacher-degree-apprenticeships.md [hunk 29]
```diff
@@ -11,7 +11,7 @@ related_content:
 navigation: 20.40
 navigation_title: If you want to do a teaching apprenticeship
 navigation_description: A teacher degree apprenticeship (TDA) lets you work at a school and earn a salary while getting a degree and qualified teacher status (QTS).
-talk_to_us: false
+talk_to_us: false
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/teacher-degree-apprenticeships.md:14]:
```suggestion
talk_to_us: false
```

File: app/views/content/train-to-be-a-teacher/if-you-dont-have-a-degree.md [hunk 30]
```diff
@@ -15,6 +15,16 @@ promo_content:
...
 navigation_title: If you do not have a degree
 navigation_description: You need a degree to get qualified teacher status (QTS). If you're not already studying for one, find out more about undergraduate degree courses.
+expander:
+  international:
+    title: undergraduate degree courses
+    text: |-
+      <p>
+      If you're a non-UK citizen who wants to study in England, you can visit the <a href="https://www.ukcisa.org.uk/">UK Council for International Student Affairs</a> for information about studying at an English university.
+      </p>      
+  teacher-degree-apprenticeships:
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/if-you-dont-have-a-degree.md:25]: @emmaachesongray This needs removing from the front matter as well - sorry, missed this yesterday!

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me
